Automattic/jetpack
# Social: Implement the new design for the admin page

This PR implements the new design for the Jetpack Social admin page.

#### Changes proposed in this Pull Request:

* Implement new design
* Add required components and split up as needed.

#### Jetpack product discussion


p6TEKc-6iM-p2

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Make sure the toggle still works (use `jetpack docker wp option get jetpack_active_modules` to ensure `publicize` gets toggled).
* Make sure the button opens the support doc for Publicize in a new page.
* Test on different sized viewports.

<table>
<tr>
	<td>Desktop</td>
	<td>Mobile</td>
</tr>
<tr>
	<td>
		<img src="https://user-images.githubusercontent.com/1713699/170038399-df9969c0-384a-4eb7-81a5-0636319f9034.png" >
	</td>
	<td>
		<img src="https://user-images.githubusercontent.com/1713699/170038425-d2aba771-cbf4-4747-81af-652fabca3941.png" >
	</td>
</tr>
</table>
